package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "anomaly-detection-dashboards",
-  "version": "1.0.0.0",
+  "version": "1.0.1.0",
   "description": "OpenSearch Anomaly Detection Dashboards Plugin",
   "main": "index.js",
   "config": {
-    "plugin_version": "1.0.0.0",
+    "plugin_version": "1.0.1.0",
```

Comment:
Does upstream OpenSearchDashboards upgrade version too? If not, we may not able to build, right?

Reply:
Yes, Dashboards has already done these bumps. The unit test and integration tests all fail until we have version parity